lib/prehog/proto/prehog/v1alpha/teleport.proto

```diff
@@ -192,6 +277,17 @@ message SubmitEventRequest {
 
     UIDiscoverStartedEvent ui_discover_started_event = 17;
     UIDiscoverResourceSelectionEvent ui_discover_resource_selection_event = 18;
+
+    UIDiscoverDeployServiceEvent ui_discover_deploy_service_event = 21;
+    UIDiscoverDatabaseRegisterEvent ui_discover_database_register_event = 22;
```

Comment:
where is 19 and 20? (don't see it as reserved)

Reply:
They were added here https://github.com/gravitational/teleport/pull/20662
Let me rebase and it should have no gaps